soroban-sdk/src/_migrating.rs

```diff
@@ -0,0 +1,223 @@
+//! # Migrating from v21 to v22
+//!
+//! 1. [`Env::register`] and [`Env::register_at`] replace [`Env::register_contract`] and [`Env::register_contract_wasm`].
+//!
+//!    [`register`] registers both native contracts previously registered with
+//!    [`register_contract`] and WASM contracts previously registered with
```

Comment:
nit: WASM->Wasm

We weren't too consistent on this, but it seems like most of the docs should now be using the 'official' abbreviation (including most of the SDK doc comments as well).

Reply:
Ah yeah, you're right, it should be `Wasm`. I get tripped on this after typing `WASM` in constants so much.
```suggestion
//!    [`register_contract`] and Wasm contracts previously registered with
```